### PR TITLE
Fixes a typo.

### LINKED ISSUE #21290: The holodeck is broken for some reason
It's loading turfs, but ONLY turfs, and none of the items/objects.

Centcom has the items and such, so it's some error with loading.

### PR DESCRIPTION
**Pull request by AnturK**
My fault for missing that in #21069

Fixes #21290

### CODE REVIEW

File: code/modules/holodeck/area_copy.dm [hunk 1]
```diff
@@ -93,7 +93,7 @@
 		X.icon_state = old_icon_state1
 
 		for(var/obj/O in T)
-			var/obj/O2 = DuplicateObject(O , perfect_copy=TRUE, newloc = X, nerf=nerf_weapons, holoitem=TRUE)
+			var/obj/O2 = DuplicateObject(O , perfectcopy=TRUE, newloc = X, nerf=nerf_weapons, holoitem=TRUE)
```
Comment (phil235) [inline code/modules/holodeck/area_copy.dm]: there's another one five lines below

Comment (AnturK) [inline code/modules/holodeck/area_copy.dm]: Thanks, why github search only shows first hit per page again ?

**Review by phil235 — APPROVED** [review]
:+1:

**Comment by optimumtact** [comment]
Dang I'm suprised it compiled

**Comment by optimumtact** [comment]
Fixes #21290

**Pull request merged**